package.json

```diff
@@ -8,6 +8,11 @@
     "prev_stable_version": "3.1.4",
     "author": "Elementor Team",
     "homepage": "https://elementor.com/",
+    "workspaces": {
+        "packages": [
+            "packages/editor/packages/*"
+        ]
+    },
```

Comment:
why do we need 2 workspaces? let's just remove the other one...

Reply:
It's a temp workspace until the package will be published